.rubocop_todo.yml

```diff
@@ -82,7 +82,7 @@ Style/GuardClause:
   Exclude:
     - 'lib/neo4j-core/query_clauses.rb'
 
-# Offense count: 56
+# Offense count: 65
```

Comment:
my guess was that no one had run `rubocop --auto-gen-config` in a while (since 2016-03-28 in fact).  A lot of metrics changed that were unrelated to what I did here, I just needed to update the metrics because I was getting caught by `AbcSize`.

Since the rest of those rules were already being muted via this file, it just updated the counts of existing violations since the previous run.